### PR TITLE
Remove unauthenticated api/applicants/submit endpoint

### PR DESCRIPTION
**Pull request by johnnymcmike**
Introduced in one of my previous PRs, the api/applicants/submit open endpoint simply pastes whatever JSON you throw at it into the database. As dev is now, it is allowing all POST requests made to this URL! Until we get authenticatd endpoints working, I have been requested to disable the unsafe endpoint. This PR does that.

### CODE REVIEW

File: src/pages/api/applicants/submit.js [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-      res.status(405).send({message: 'Only POST requests at this endpoint, please'})
-      return;
-    }
-    //not sure how to sanity-check that these properties exist in the request body, but that should be done here
-    const newref = Firestore_Database.collection('applicants').doc(`${req.body.firstName}-${req.body.lastName}-${req.body.email}`.toLowerCase());
-    await newref.set(req.body);
-    res.status(200).send()
-  }
\ No newline at end of file
+// import { firestoreApp  } from '../../../database';
```
Comment (Sanjay-B) [inline src/pages/api/applicants/submit.js:1]: Nitpick: Would appreciate a comment explaining why this is commented out. Even though it's inherently obvious to us, it won't be for those who have no knowledge about this PR.

**Review by Sanjay-B — CHANGES_REQUESTED** [review]
Overall looks good. All PRs labeled under `security` needs a security opinion / review before merging. So, I've included @teamrocket77 as a reviewer.

File: src/pages/api/applicants/submit.js (hunk 1 above)
Comment (teamrocket77) [reply src/pages/api/applicants/submit.js:1]: I agree I think a comment would do others some good.

**Review by teamrocket77 — CHANGES_REQUESTED** [review]
Requested: Comments explaining why this is commented out

File: src/pages/api/applicants/submit.js (hunk 1 above)
Comment (johnnymcmike) [reply src/pages/api/applicants/submit.js:4]: Good point, comment added in latest commit

**Review by teamrocket77 — COMMENTED** [review]
LGTM

**Review by teamrocket77 — APPROVED** [review]
LGTM

**Review by Sanjay-B — APPROVED** [review]
Looks good to me.

**Comment by Sanjay-B** [comment]
Reviewing conditions have been met. Merging into ``dev``. Thank you for correcting this. @johnnymcmike cc: @PremKSU

**Pull request merged**